### ISSUE TITLE
Add IDs for key purposes

### ISSUE THREAD

**Issue by xipki** [cose-wg/CBOR-certificates#141]
In the current draft (version 7), only a small subset of the well-known key purposes are assigned. Since key purposes are widely used in the X.509 certificates, please assign as far as possible ID to such key purposes. For me, at least for the following non-private key purposes: 1.3.6.1..5.7.3.x.

A list of the OIDs are available under
http://oid-info.com/get/1.3.6.1.5.5.7.3

Thanks

**Comment by xipki** [comment]
Thank you.
as opensource pki developer, I need the following CMC key purposes:
- RFC 6402 Certificate Management over CMS (CMC) Updates
  - 1.3.6.1.5.5.7.3.27 id-kp-cmcCA (RFC 
  - 1.3.6.1.5.5.7.3.28 id-kp-cmcRA
  - 1.3.6.1.5.5.7.3.29 id-kp-cmcArchive
- RFC 9480 Certificate Management Protocol (CMP) Updates
  - 1.3.6.1.5.5.7.3.27 id-kp-cmcKGA

**Comment by xipki** [comment]
Yes, it is id-kp-cmKGA with 32. I fixed it in my previous comment.

**Comment by highlunder** [comment]
Added in commit e1da2e9! Closing!
